### PR TITLE
Can't open URL from chat in web browser [#3600]

### LINKED ISSUE #3600: Can't open URL from chat in web browser
### User Story

As a user I want to open sent/received URLs in web browser on my phone

### Description

*Type*: Bug-Bounty

*Summary*: Nothing happens when trying to open URL from a chat in web browser

#### Expected behavior
URL is opened in web browser 

#### Actual behavior
Application chooser dialog is hidden. URL is not opened

### Reproduction
- Open Status
- Navigate to a chat (one-to-one, group or public)
- Send some URL in the chat
- Tap the URL
- Tap 'Open in Web browser'

### Additional Information
* Status version: 0.9.10-776-g9cae0a5d+ (nightly build 277 from March 16)
* Operating System: 
Real device LG V20, Android 7.0
Real device iPhone 8+, iOS 11.1.1

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #3600

### CODE REVIEW

File: src/status_im/utils/http.cljs [hunk 1]
```diff
@@ -56,3 +55,6 @@
        (.catch (or on-error
                    (fn [error]
                      (utils/show-popup "Error" (str error))))))))
+
+(defn match-url [url]
```
Comment (jeluard) [inline src/status_im/utils/http.cljs]: Maybe we could improve the name of this fn? Like `normalize-url` ? (I know it was there already :))

**Review by jeluard — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

File: src/status_im/utils/http.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/utils/http.cljs]: yes, i agree, `normalize-url` sounds better, thanks

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/3swKCM for branch PR-4377

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/xPNCtE for branch PR-4377

<blockquote><div><strong><a href="https://i.diawi.com/xPNCtE">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/pKfmGG for branch PR-4377

<blockquote><div><strong><a href="https://i.diawi.com/pKfmGG">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Pull request merged**